.drone/drone.jsonnet

```diff
@@ -373,8 +373,8 @@ local manifest(apps) = pipeline('manifest') {
     steps: [
       make('check-drone-drift', container=false) { depends_on: ['clone'] },
       make('check-generated-files', container=false) { depends_on: ['clone'] },
-      make('test', container=false) { depends_on: ['clone'] },
-      run('clone-main', commands=['cd ..', 'git clone $CI_REPO_REMOTE loki-main', 'cd -']),
+      run('clone-main', commands=['cd ..', 'git clone $CI_REPO_REMOTE loki-main', 'cd -']) { depends_on: ['clone'] },
+      make('test', container=false) { depends_on: ['clone', 'clone-main'] },
```

Comment:
`clone-main` changes to another directory and that changes the directory for _all jobs in this pipeline_ causing concurrent jobs to run. This change makes sure everything is cloned and ready before we try to run tests.